### DISCUSSION TITLE
Unable to see Dashboards with any user.

### DISCUSSION THREAD

**Discussion by golu-dailype** [getredash/redash#6946] [Q&A] [answered]
I connected a new data source today onto my Redash and ever since then on any user group I am unable to see the dashboards that I had created.
I can see the Dashboard count in settings as well as using sql queries on my EC2 but unable to access them from any user. Please help.
![image](https://github.com/getredash/redash/assets/140686223/21c8d65a-1df8-4fea-b4e2-93db8b9d1a51)

**Comment by justinclift** [comment]
That doesn't sound good at all.  ~~Which version of Redash are you using?~~ Just noticed the `10.1.0` in the version number there.

Also, if all else fails, how recent is your last backup?  Just in case. :smile:

**Comment by justinclift** [comment]
Since you're using Redash 10.x things shouldn't be too far wrong.

Are you familiar enough with ssh that you can log into your Redash EC2 vm to check the Docker log files?

In theory (!), any error that's being generated by the new data source should be showing up in there.

So, if you can get the Docker log files, then it should be possible to figure out what's going wrong.  And from there we can probably figure out how to fix it. :smile:

**Reply by golu-dailype** [reply, marked as answer]
So we found the issue,
While adding a new data source we replaced the old data source for a while.
And the Data source table in Redash has its own foreign ID key to queries and dashboard tables.
The foreign key got removed from all tables making the queries and dashboards not visible to any of the users.
We used the: update queries set data_source = <new data source id> where name = <query name>;
Although my understanding of the exact issue might not be accurate but the above fix helped me get our Redash back to the way it was.
Thank you for the help!

**Reply by justinclift** [reply]
You're welcome, glad you got it solved. :smile:
